conf/test.config

```diff
@@ -20,10 +20,6 @@ params {
     max_time   = '6.h'
 
     // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
-
-    // Genome references
-    genome = 'R64-1-1'
+    input = "$baseDir/tests/assets/test_samplesheet.csv"
+    output = "${projectDir}/tests/results"
```

Comment:
The parameter name is `--outdir` instead of `--output`.

Also, normally I have been leaving the `outdir` parameter as unset for the test profile, so that it must be provided on the command-line when running nextflow with the test profile:

```
nextflow run phac-nml/nf-core-staramr -profile test,docker --outdir results
```

Could you remove setting this parameter here?

Reply:
Sounds good!